sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -98,8 +98,8 @@
 
 /**
  * An unbounded source and a sink for <a href="http://kafka.apache.org/">Kafka</a> topics.
- * Kafka version 0.9 and above are supported. It's supported to replace with Kafka client 0.10
- * for external authentication.
+ * Kafka version 0.9 and 0.10 are supported. To specify Kafka client 0.10 in pom.xml
```

Comment:
Something like : "If you need a specific version of Kafka client (e.g. 0.9 for 0.9 servers, or 0.10 for security features), specify explicit kafka-client dependency."